src/mmw/js/src/analyze/views.js

```diff
@@ -466,6 +469,104 @@ var PointSourceTableView = Marionette.CompositeView.extend({
     }
 });
 
+var CatchmentWaterQualityTableRowView = Marionette.ItemView.extend({
```

Comment:
With this fifth tab, I found this file to be awkwardly large - maybe we should be splitting off each view into its own file now?